godtools/Views/Cells/LanguageTableViewCell.swift

```diff
@@ -21,8 +21,11 @@ class LanguageTableViewCell: UITableViewCell {
     var cellDelegate: LanguageTableViewCellDelegate?
     var language: Language? {
         didSet {
-            languageExists(language!.shouldDownload)
-            languageLabel.text = language!.localizedName()
+            guard let language = language else {
+                return
```

Comment:
Just a question, shouldn't you set language(Exists) to false?

Reply:
ah, si!! good catch!